lenskit-cli/src/main/java/org/lenskit/cli/util/Logging.java

```diff
@@ -103,20 +119,18 @@ public static void configureLogging(Namespace options) {
             filePat.setPattern(FILE_PATTERN);
             filePat.start();
             fileOutput.setEncoder(filePat);
+
+            ThresholdFilter filter = new ThresholdFilter();
+            filter.setContext(context);
+            filter.setLevel(logFileLevel.toString());
+            filter.start();
+            fileOutput.addFilter(filter);
+
             fileOutput.start();
+
             root.addAppender(fileOutput);
-            root.setLevel(logLevel);
-            if (!debug) {
-                ThresholdFilter filter = new ThresholdFilter();
-                filter.setContext(context);
-                filter.setLevel("INFO");
-                filter.start();
-                console.addFilter(filter);
-            }
-        } else if (debug) {
-            root.setLevel(Level.DEBUG);
-        } else {
-            root.setLevel(Level.INFO);
+
+            root.setLevel(logFileLevel);
```

Comment:
II might be wrong here, but is seems like the rood.setLevel on 110 and 133 can both be reached. That seems strange, but not necessarily wrong, to me (for lack of knowing the logging configuration framework well enough to know better).